docs/quickstart-myst-documents.md

```diff
@@ -98,8 +98,8 @@ Unfortunately, this way of including this information is not easily machine-read
 
 A tutorial to evolve markdown documents and notebooks into structured data
 
-**Authors:** Rowan Cockett<sup>1,2</sup> \
-**Affiliations:** <sup>1</sup>Executable Books, <sup>2</sup> Curvenote \
+**Authors:** Rowan Cockett <sup>1,2</sup> \
+**Affiliations:** <sup>1</sup> Executable Books, <sup>2</sup> Curvenote \
```

Comment:
In the according docs screenshot, the spacing could be updated as well :
<img width="450" alt="image" src="https://user-images.githubusercontent.com/44469195/218872729-8e7bb052-6542-4301-b9bc-280103f1b7b0.png">